src/Blueprint.h

```diff
@@ -357,10 +340,18 @@ namespace snowcrash {
             UndefinedElement = 0, // Unknown
             CategoryElement,      // Group of other elements
             CopyElement,          // Human readable text
+            AssetElement,         // Asset of API description
             ResourceElement,      // Resource
             DataStructureElement  // Data Structure
         };
 
+        /** Role of an element */
+        enum Role {
```

Comment:
This is Asset-specific not Element specific attribute. Please move it appropriately.